Bottom: 157e6a7f961919a389353525a8a276e1ab693815
Top:    147cbb292a19cdaa49cb27217c9c1ae686469eef
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-12 01:20:27 +0000

Refresh of remove-invalidate_lr-from-the-

---

diff --git a/src/hid/gcode/gcode.c b/src/hid/gcode/gcode.c
index bbc1d53..c3c2e4e 100644
--- a/src/hid/gcode/gcode.c
+++ b/src/hid/gcode/gcode.c
@@ -1040,7 +1040,6 @@ HID gcode_hid = {
   gcode_get_export_options,
   gcode_do_export,
   gcode_parse_arguments,
-  0 /* gcode_invalidate_wh */ ,
   0 /* gcode_invalidate_lr */ ,
   0 /* gcode_invalidate_all */ ,
   gcode_set_layer,
